src/__tests__/__snapshots__/Organization-test.js.snap

```diff
@@ -603,7 +603,7 @@ exports[`Organization editor renders correctly 1`] = `
                       <img
                         alt=""
                         className="img-fluid"
-                        src="http://uploads.participedia.xyz.s3-website-us-east-1.amazonaws.com/uis_favicon_0.png"
+                        src="http://uploads.participedia.xyz.s3-website-us-east-1.amazonaws.comuis_favicon_0.png"
```

Comment:
Here it looks like your config is missing a slash, so the host and path are getting run together.